### PR TITLE
Global updates

### PR DESCRIPTION
**Pull request by paulgildea**
Adding the Global Token definitions. Remapped so the demo would be functional.

### CODE REVIEW

File: src/demo/fluentui.json [hunk 1]
```diff
@@ -173,7 +246,7 @@
 		},
 		"Illustration": {
 			"Corner": {
-				"Radius": { "aliasOf": "Global.Corner.Radius.ExtraLarge" }
+				"Radius": { "aliasOf": "Global.Corner.Radius.X-Large" }
 			}
 		},
```
Comment (TravisSpomer) [inline src/demo/fluentui.json:251]: It looks like they may have removed this entirely...? I don't see it anywhere, but it's impossible to find stuff in Figma...

**Review by TravisSpomer — CHANGES_REQUESTED** [review]

File: src/demo/fluentui.json [hunk 2]
```diff
@@ -5,27 +5,99 @@
...
 			},
 			"Grey": {
 				"generate": { "type": "lightness0to100by2", "value": "grey" }
 			},
 			"White": { "value": "#ffffff" },
-			"Black": { "value": "#000000" }
+			"Black": { "value": "#000000" },
+			"Hyperlink": {"value": "#FFFF00"},
+			"Disabled": {"value": "#3FF23F"},
+			"Selected": {"value": "#1AEBFF"},
```
[the remark this answers is not in the document]
Comment (paulgildea) [reply src/demo/fluentui.json:33]: Yeah, they are placeholders for now. Here's how @miroslavstastny is thinking about approaching from the theme perspective: https://github.com/miroslavstastny/fluentui/blob/a8d647fa29aac052a212a1049a2bcb4e084825d8/rfcs/high-contrast-theme.md

So we might need a platform agnostic way to reference them, or have a pipeline way to address high contrast

File: src/demo/fluentui.json [hunk 3]
```diff
@@ -5,27 +5,99 @@
 	"Global": {
 		"Color": {
 			"Brand": {
-				"generate": { "type": "lightness0to100by2", "value": "hsl(206, 100%, 50%)" }
+				"Shade": {
+					"60": {"value": "#092C47"},
```
[the remark this answers is not in the document]
Comment (paulgildea) [reply src/demo/fluentui.json:9]: I'll follow up with design on this one. Yeah it's interesting that it isn't symmetrical.

File: src/demo/fluentui.json [hunk 4]
```diff
@@ -5,27 +5,99 @@
...
+					"50": {"value": "#043862"},
+					"40": {"value": "#004578"},
+					"30": {"value": "#004C87"},
+					"20": {"value": "#005A9E"},
+					"10": {"value": "#106EBE"}
+				},
+				"Primary": {"value": "#106EBE"},
+				"Tint":{
+					"10": {"value": "#2899F5"},
```
[the remark this answers is not in the document]
Comment (paulgildea) [reply src/demo/fluentui.json:18]: That sounds like a plan. I don't have a super strong preference. I believe the Shade and Tint concepts are hold overs from the Fabric color values. Most likely to help product teams make the transition easier.

File: src/demo/fluentui.json [hunk 5]
```diff
@@ -5,27 +5,99 @@
...
 			"Grey": {
 				"generate": { "type": "lightness0to100by2", "value": "grey" }
 			},
 			"White": { "value": "#ffffff" },
-			"Black": { "value": "#000000" }
+			"Black": { "value": "#000000" },
+			"Hyperlink": {"value": "#FFFF00"},
+			"Disabled": {"value": "#3FF23F"},
+			"Selected": {"value": "#1AEBFF"},
+			"Shared": {
```
[the remark this answers is not in the document]
Comment (paulgildea) [reply src/demo/fluentui.json:34]: I'll pull the latest changes from main for this one. 👍

File: src/demo/fluentui-web.json [hunk 6]
```diff
@@ -0,0 +1,143 @@
+{
```
[the remark this answers is not in the document]
Comment (paulgildea) [reply src/demo/fluentui-web.json:1]: Most likely. Will remove it

File: src/demo/fluentui.json [hunk 7]
```diff
@@ -5,27 +5,99 @@
...
+			}
 		},
 		"Stroke": {
 			"Width": {
-				"0": { "value": 0 },
-				"100": { "value": 1 }
+				"Thin": {"value": 1},
+				"Thick": {"value": 2},
+				"Thicker": {"value": 3, "platform": {"ios": {"value": 4}}},
+				"Thickest": {"value": 4, "platform": {"ios": {"value": 6}}}
```
[the remark this answers is not in the document]
Comment (paulgildea) [reply src/demo/fluentui.json:91]: 👍

File: src/demo/fluentui.json [hunk 8]
```diff
@@ -5,27 +5,99 @@
...
 		},
 		"Corner": {
 			"Radius": {
 				"None": { "value": 0 },
 				"Small": { "value": 2 },
 				"Medium": { "value": 4 },
-				"Large": { "value": 6 },
-				"ExtraLarge": { "value": 8 },
+				"Large": { "value": 6, "platform": {"ios": {"value": 8}} },
+				"X-Large": { "value": 8, "platform": {"ios": {"value": 12}} },
```
[the remark this answers is not in the document]
Comment (paulgildea) [reply src/demo/fluentui.json:100]: Yeah... I saw this value and had a similar reaction to having the dash. I'm going to change it back to ExtraLarge and ask Design to rename it in the Figma.

File: src/demo/fluentui.json (hunk 1 above)
Comment (paulgildea) [reply src/demo/fluentui.json:251]: Yeah, I'll verify to make sure.

File: src/demo/fluentui.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (TravisSpomer) [reply src/demo/fluentui.json:33]: Yeah, I've got high contrast support as #34; Windows hasn't needed anything additional from the pipeline yet, but I've got that issue to track whatever requests we get from there. I assume it'll be something along the lines of allowing people to use those web high contrast color slots as predefined "aliases."

[the remark this answers is not in the document]
Comment (paulgildea) [reply src/demo/fluentui.json:33]: I'll remove them for now.

**Pull request merged**